test/e2e/specs/plugins-api.test.js

```diff
@@ -17,17 +22,28 @@ describe( 'Using Plugins API', () => {
 		await deactivatePlugin( 'gutenberg-test-plugin-plugins-api' );
 	} );
 
-	it( 'Should open plugins sidebar using More Menu item and render content', async () => {
-		await toggleMoreMenuItem( 'My title plugin' );
+	describe( 'Post Status Info', () => {
+		it( 'Should render post status info inside Document Setting sidebar', async () => {
+			await ensureDocumentSettingsSidebarIsOpened();
 
-		const pluginSidebarContent = await page.$eval( '.edit-post-sidebar', ( el ) => el.innerHTML );
-		expect( pluginSidebarContent ).toMatchSnapshot();
+			const pluginPostStatusInfoText = await page.$eval( '.my-post-status-info-plugin', ( el ) => el.innerText );
```

Comment:
I think   we should improve the selector to `.edit-post-post-status .my-post-status-info-plugin' to test that it was rendered "inside" right now if we render outside of Document Setting sidebar the test will pass.